plugins/modules/dcnm_interface.py

```diff
@@ -1214,6 +1339,184 @@
+          enable_netflow: false                     # optional, flag to enable netflow, default is false
+          mode: port_channel_aa                     # choose from [port_channel_aa], default is "port_channel_aa"
+
+          peer1_cmds:                               # optional, freeform config, default is []
+            - ip arp inspection trust
+          peer2_cmds:                               # optional, freeform config, default is []
+            - ip arp inspection trust
+
+- name: Delete AA FEX interfaces
+  cisco.dcnm.dcnm_interface:
+    check_deploy: True
+    fabric: "{{ ansible_svi_fabric }}"
+    state: deleted                        # only choose form [merged, replaced, deleted, overridden, query]
+    config:
+      - name: vpc151                      # should be of the form vpc<id>
+        switch:
+          - "{{ ansible_switch1 }}"       # provide the switch where to deploy the config
+
+
+- name: Overide AA FEX interface with a new one
+  cisco.dcnm.dcnm_interface:
+    check_deploy: true
+    fabric: "{{ ansible_svi_fabric }}"
+    state: overridden                               # only choose form [merged, replaced, deleted, overridden, query]
+    config:
+      - name: vpc151                                # should be of the form vpc<id>
+        type: aa_fex                                # choose from this list [pc, vpc, sub_int, lo, eth, svi, st_fex, aa_fex]
+        switch:
+          - "{{ ansible_switch1 }}"                 # provide the switch information where the config is to be deployed
+        deploy: true                                # choose from [true, false]
+        profile:
+          description: "AA FEX interface 151"       # optional, description of FEX interface, default is ""
+          peer1_members:                            # optional, member interfaces, default is []
+            - e1/10
+          peer2_members:                            # optional, member interfaces, default is []
+            - e1/10
+          mtu: "jumbo"                              # optional, MTU for the interface, default is "jumbo"
+          peer1_po_description: "PC 151 for AA FEX" # optional, description of PC interface, default is ""
+          peer2_po_description: "PC 151 for AA FEX" # optional, description of PC interface, default is ""
+          peer1_cmds:                               # optional, freeform config, default is []
+            - no shutdown
+          peer2_cmds:                               # optional, freeform config, default is []
+            - no shutdown
+          admin_state: true                         # Flag to enable/disable FEX interface.
+          enable_netflow: false                     # optional, flag to enable netflow, default is false
+          mode: port_channel_aa                     # choose from [port_channel_aa], default is "port_channel_aa"
+
+# STRAIGHT-THROUGH FEX INTERFACES
+
+- name: Create ST FEX interfaces including optional parameters
```

Comment:
Here also, let's use straight-through instead of ST

Reply:
Sure